apps/web/src/components/forms/edit-document/add-subject.action.ts

```diff
@@ -9,16 +9,18 @@ export type CompleteDocumentActionInput = TAddSubjectFormSchema & {
   documentId: number;
 };
 
-export const completeDocument = async ({ documentId, email }: CompleteDocumentActionInput) => {
+export const completeDocument = async ({ documentId, meta }: CompleteDocumentActionInput) => {
   'use server';
 
   const { user } = await getRequiredServerComponentSession();
 
-  if (email.message || email.subject) {
+  if (meta.message || meta.subject) {
     await upsertDocumentMeta({
       documentId,
-      subject: email.subject,
-      message: email.message,
+      subject: meta.subject,
+      message: meta.message,
+      timezone: meta.timezone,
+      dateFormat: meta.dateFormat,
     });
   }
```

Comment:
Doesn't this mean we would never capture our timezone changes unless we filled out a custom email template as well? 👀

Reply:
Oopsies! I will update the if condition  😶

Reply:
Currently the code is like this,

```ts
if (meta.message || meta.subject || meta.dateFormat || meta.timezone) {
    await upsertDocumentMeta({
      documentId,
      subject: meta.subject,
      message: meta.message,
      timezone: meta.timezone,
      dateFormat: meta.dateFormat,
    });
  }
```

how about if we update it to make it to 

```ts
if (meta.message || meta.subject) {
    await upsertDocumentMeta({
      documentId,
      subject: meta.subject,
      message: meta.message,
    });
  }
  
if (meta.dateFormat || meta.timezone) {
    await upsertDocumentMeta({
      documentId,
      timezone: meta.timezone,
      dateFormat: meta.dateFormat,
    });
  }
```
and then update the `upsertDocumentMeta` function?